### ISSUE TITLE
Lantmäteriet map layers in Leaflet

### ISSUE THREAD

**Issue by ewels** [ewels/hemnet-commuter#11]
Lantmäteriet has an open-data system with free accounts and API keys: https://opendata.lantmateriet.se/

I think that this could be used to render Lantmäteriet maps directly in Hemnet Commuter, which would be awesome.

GitHub repo with demonstration code here: https://github.com/fredriklindmark/lantmateriet_wmts - need to think about how this would work with angular-leaflet-directive which I'm using: http://angular-ui.github.io/ui-leaflet/#!/

Whilst doing this, could also even think about doing the same with Google Maps and have multiple layers to choose from if we really want to make it pro. We already use Google Maps API keys.

**Comment by ewels** [comment]
Done in fcf70d9f0038980c9f7659abefd8a86db3c6e9a7, though it sadly isn't as awesome as I was hoping for. Gives a topo map, but I can't figure out how to have the topo with tomt borders, or satellite (with tomt borders).

For googlers / future me - the topo uses WMTS but the other Lantmäteriet maps use WMS. The latter is possible in Leaflet, see https://leafletjs.com/examples/wms/wms.html for a description and [here for an example](http://angular-ui.github.io/ui-leaflet/examples/0000-viewer.html#/layers/wms-with-different-projection-example).

I got this to work with Lantmäteriet but couldn't display any particularly useful maps with it. I got what I guess were height point markers or something, but no pretty maps. I could have maybe dug into satellite but I could only see historical and not current day. Most importantly, I couldn't see data for the tomt gränser anywhere, which is what I'm usually most interested in.

So, partial success :)
